docs/content/Configuration/SLS.mdx

```diff
@@ -0,0 +1,127 @@
+---
+title: Semantic Layer Sync
+permalink: /semantic-layer-sync
+category: Configuration
+menuOrder: 5
+---
+
+Semantic Layer Sync synchronizes the [data model][ref-data-model]
+of a semantic layer with datasets in BI tools. It's the easiest way to
+connect a BI tool to the semantic layer.
+
+Semantic Layer Sync programmatically connects a BI tool to Cube via the
+[SQL API][ref-sql-api] and creates BI-specific entities that correspond to
+entities within the data model, e.g., cubes, views, measures, dimensions:
+
+| Cube                        | BI tool                   |
+| --------------------------- | ------------------------- |
+| Deployment                  | Database                  |
+| Cube, view                  | Dataset, table            |
+| Measure, dimension, segment | Column, dimension, metric |
+
+## Supported tools
+
+Semantic Layer Sync supports the following BI tools:
+
+* [Apache Superset][ref-superset]
+* [Metabase][ref-metabase]
+* [Preset][ref-preset]
+
+<InfoBox>
+
+We're working on bringing support for other BI tools. Please
+[reach out to us](https://forms.gle/eEit9Pa7ZXSoFxb86) if you're interested
+in using Semantic Layer Sync with other BI tools.
+
+</InfoBox>
+
+## Configuration
+
+Semantic Layer Sync is configured via the [semanticLayerSync][ref-config-sls]
+option in the `cube.js` configuration file.
+
+<SuccessBox>
+
+Cube Cloud [has a dedicated page][ref-workspace-sls] to manage Semantic Layer
+Sync: configure syncs, run them, and view synchronization history.
+
+</SuccessBox>
+
+### Prerequisites
+
+The following configuration is required for Semantic Layer Sync to work:
+
+* Semantic Layer Sync runs on a [refresh worker][ref-refresh-worker].
+You have to have a refresh worker as part of your Cube deployment.
```

Comment:
@keydunov @vasilev-alex I think we should emit a warning to logs if Cube runs in production mode and Semantic Layer Sync configuration is defined but the Cube instance is not a refresh worker. WDYT?